tests/TrackPageviewTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+use Pirsch\Facades\Pirsch;
+use Pirsch\Http\Middleware\TrackPageview;
+
+it('handles request', function () {
+    Pirsch::shouldReceive('track')
+        ->once();
+
+    Route::middleware(TrackPageview::class)
+        ->get('/', fn () => 'Hello World');
+
+    $this->get('/');
+
+    expect(true)->toBeTrue();
```

Comment:
Isn't it cleaner to use `throwsNoExceptions()` instead?
https://pestphp.com/docs/exceptions

Reply:
As you wish!

Reply:
Actually, those tests already have an assertion, and with `throwsNoExceptions` it lead to this error:
```
 RISKY  Tests\TrackPageviewTest > it handles request                                                                                                                                        
  This test is not expected to perform assertions but performed 1 assertion
```
... so I just removed the `except..`